.github/workflows/verifyVersion.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+module.exports = async (github, context) => {
+  const issue = context.payload.issue;
+
+  // Ignore issues using upgrade template (they use a special label)
+  if (issue.labels.find(label => label.name == 'Type: Upgrade Issue')) {
+    return;
+  }
+
+  // Extract RN version number from the issue body, if it exists
+  const getReactNativeVersionIfExists = issue => {
+    if (!issue || !issue.body) return;
+    const rnVersionRegex =
+      /React Native Version[\r\n]+[0-9]{1,2}\.[0-9]{1,2}\.[0-9]{1,2}/;
```

Comment:
I'll address the three points separately:
1. Multiple versions/unexpected formatting in versions - I definitely came across a few cases like this, too, when looking for examples. I think we can improve the situation here by being a bit more directive with the description/placeholder provided in the new issue template - i.e. if we say something like "What is the latest version of react-native that this issue reproduces on? Please only list the highest version you tested," that might reduce the occurrence of unexpected formatting. The script will tag anything it's unable to parse with 'Needs: Version Info', which will hopefully lead authors to correct their description. Thoughts?
2. I *think* the current logic will actually work correctly for pre-releases (sort of by accident), since I check whether the actual reported minor number < latest - 2 to figure out if the version is unsupported, and prereleases will always have higher version numbers than the latest. With that said, this will fail for major version checks (since that is just a != check). This regex should still match pre-releases since the core format of x.x.x is the same (and prereleases also have -rc.x at the end). However, I can definitely make this more robust and support this "officially"
3. Are those always reported as "nightly?" I can add handling to ignore those for version checks.